api/v1alpha1/backupstorage_types.go

```diff
@@ -74,3 +78,21 @@ type BackupStorageList struct {
 func init() {
 	SchemeBuilder.Register(&BackupStorage{}, &BackupStorageList{})
 }
+func (b *BackupStorage) UpdateNamespacesList(namespace string) {
+	if b.Status.Namespaces == nil {
+		b.Status.Namespaces = make(map[string]bool)
+	}
+	b.Status.Namespaces[namespace] = true
```

Comment:
we only add the namespaces to the Status but not delete them, it means we may end up in the situation when a namespace is listed in the Status but not even exists in the cluster. Is it an intended trade-off?